Clean up Grants page and refs

DrW3RK opened the pull request:
The grants page has to be made generic, providing information on all the funding sources available in the ecosystem

DrW3RK committed:
Clean up Grants page and refs

DrW3RK committed:
rephrase description of ref to grants page

alxs commented on docs/general/grants.md:
```diff
@@ -1,20 +1,17 @@
+regarding requirements, the application process, deliveries, etc., can be found in the
 [Grants Program README](https://github.com/w3f/Open-Grants-Program). For guidance, there is also a list of
 [previously accepted applications](https://github.com/w3f/Grants-Program/blob/master/docs/accepted_grant_applications.md)
 and a list of [frequently asked questions](https://github.com/w3f/Grants-Program/blob/master/docs/faq.md).
```
```suggestion
Web3 Foundation offers grants for open source software development and research around Substrate,
Polkadot, Kusama and ink!. Applications and deliveries are tracked transparently on GitHub. Information 
regarding requirements, the application process, deliveries, etc. can be found in the
[Grants Program README](https://github.com/w3f/Grants-Program#web3-foundation-grants-program). For guidance, there is also a list of
[previously accepted applications](https://github.com/w3f/Grants-Program/blob/master/docs/accepted_grant_applications.md#accepted-grant-applications-)
and a list of [frequently asked questions](https://github.com/w3f/Grants-Program/blob/master/docs/faq.md).
```

alxs commented:
The list of alternative funding sources has been updated, see https://github.com/w3f/Grants-Program#rocket-alternative-funding-sources

DrW3RK committed:
Update docs/general/grants.md

Co-authored-by: Aleixo Sanchez <15819210+alxs@users.noreply.github.com>

DrW3RK committed:
Update grants.md

DrW3RK commented:
Added a link to the alternative funding sources as well.

alxs commented:
The list of other grants programs is already in this page, but it's outdated.

The whole "Alternative Funding Sources" section, which is copied from our readme into this page, has been updated. I suggest just copy-pasting the contents, maybe you want to change one or two things.

DrW3RK committed:
Add info note and link to grants repo doc

DrW3RK commented:
@alxs makes sense. Added a note (as info) linking to the grants section on alternative section.

alxs commented on docs/general/grants.md:
```diff
@@ -1,26 +1,31 @@
+
+Check the 
+[alternative funding sources](https://github.com/w3f/Grants-Program/blob/master/README.md#rocket-alternative-funding-sources) 
+section on Web3 Foundation grants repo for comprehensive information.
```
```suggestion
section on the Web3 Foundation grants repo for comprehensive information.
```

DrW3RK committed:
remove Darwinia

No active grants program at Darwinia. The previous link https://docs.darwinia.network/developers/dev-bounty#grant-program was taken down and the github link has not been updated for a while

DrW3RK committed:
Update docs/general/grants.md

Co-authored-by: Aleixo Sanchez <15819210+alxs@users.noreply.github.com>

DrW3RK commented:
Approve if this PR looks good. Will merge right after

alxs commented:
Actually. Sorry, wait a sec

alxs commented on docs/general/grants.md:
```diff
@@ -43,10 +48,16 @@ If you are interested in obtaining KSM for building or research, you can apply t
 
-- [Moonbeam Grants Program](https://moonbeam.network/community/grants/)
-- [Edgeware Grants and Bounties](https://github.com/edgeware-builders/construction-projects)
+- [Moonbeam Grants Program](https://moonbeam.foundation/grants/)
```
```suggestion
- [Darwinia Grants Program](https://github.com/darwinia-network/collaboration/blob/master/grant/README.md#grant-program)
- [Moonbeam Grants Program](https://moonbeam.foundation/grants/)
```

alxs commented:
The fact that they haven't updated their GitHub in a while doesn't mean it's not active. They have 1 open application at the moment ;) https://github.com/darwinia-network/collaboration/pulls But feel free to double check with them if they're no longer accepting applications

DrW3RK committed:
Update docs/general/grants.md

Co-authored-by: Aleixo Sanchez <15819210+alxs@users.noreply.github.com>

DrW3RK commented:
They took down the grants page from their docs which was the recent addition to the Wiki. We can have this github link stay in the Wiki (for now)
